internal/client/blueprints_test.go

```diff
@@ -1229,3 +1229,56 @@ func TestBlueprintChangeV1(t *testing.T) {
 	require.NotNil(t, bp, "GET blueprint change failed: missing blueprint")
 	assert.Equal(t, bp.Version, "0.0.1")
 }
+
```

Comment:
The (loose) idea in the blueprint testing is to test the same things as JSON and TOML to make sure they return the same result/error and it's easier to compare things if the content of the blueprint is the same in the TOML as it is in the JSON.

Reply:
So my tests are fine or how should they be changed? 🤔

Reply:
... when I moved the implementation to `internal/blueprint/blueprint.go` probably the tests should also go to `internal/blueprint/blueprint_test.go`?